code/game/mecha/mech_fabricator.dm

```diff
@@ -71,6 +72,14 @@
 								"IPC Components",
 								"Misc"
 								)
+	var/list/combat_parts = list(
```

Comment:
Unused variable

Reply:
ah yeah thats just a holdover from it being an old branch that i had to update